portfolio/src/main/webapp/galleryScript.js

```diff
@@ -0,0 +1,20 @@
+// Functions for the slideshows in Gallery
+var slideIndex = 1;
+displaySlide(slideIndex);
+
+function moveSlide(n) {
+  displaySlide(slideIndex += n);
+}
+
+function displaySlide(n) {
+  var slides = document.getElementsByClassName("EuropeSlideshow");
```

Comment:
nit: if the slides are never expected to change, you can declare this in the top-level scope (alongside `slideIndex`)

(this will make the above suggestion easier too)

Reply:
Got it!